packages/api/middlewares/authentication.js

```diff
@@ -65,6 +66,14 @@ module.exports = () => {
           error = true;
         }
       } else {
+        if (shortApiKey == true) {
+          const propertyName = req.params?.propertyName;
+          if (propertyName != data.propertyName) {
+            error = true;
+            res.status(400).json({ message: "Invalid APIkey for property" });
```

Comment:
If I get this correct, this block is validating the  requested property with JWT info, in that case the status should be 403, and the message would be access denied. and log it in console that unauthorised access detected property name doesn't match  -  @soumyadip007

Reply:
Resolved , please check

Reply:
looks good.